arquillian-chameleon-deployments/arquillian-chameleon-gradle-build-deployment/src/test/java/org/arquillian/container/chameleon/deployment/gradle/GradleBuildAutomaticDeploymentTest.java

```diff
@@ -0,0 +1,23 @@
+package org.arquillian.container.chameleon.deployment.gradle;
+
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.shrinkwrap.api.Archive;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class GradleBuildAutomaticDeploymentTest {
+
+    @Test
+    public void should_build_archive() {
+        GradleBuildAutomaticDeployment gradle = new GradleBuildAutomaticDeployment();
+        Archive<?> archive = gradle.build(new TestClass(TestCaseClass.class));
+
+        Assert.assertTrue(archive.contains("/WEB-INF/classes/org/arquillian/example/helloworld/GreetingService.class"));
```

Comment:
we are using assertj in tests, can you update this assertion to be assertj fashion?

Reply:
done.